src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,85 @@
+import classNames from 'classnames';
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[],
+  selectedTodoId?: number | null,
```

Comment:
```suggestion
  selectedTodoId?: number | null,
```
if property is indicated as optional, you don't need it to be `null`, apply either